scripts/release_scripts/generate_release_updates.py

```diff
@@ -159,8 +159,16 @@ def create_group_for_next_release():
         'the release co-ordinator & QA Lead for next release. '
         'Please follow the release process doc: '
         '[Add link to release process doc] to ensure the release '
-        'follows the schedule. Thanks!\n' % (
-            release_constants.RELEASE_ROTA_URL))
+        'follows the schedule. Thanks!\n\n'
+        '3. Send the following message to oppia-dev:\n\n'
+        'Hi all, We have completed the release for current month. '
+        'This is a reminder to fill in the job requests here: '
+        '%s if you are planning '
+        'to run your job in the next release. '
+        'Please fill in the requests by [Add a deadline which is 7 '
```

Comment:
Optional: do we want to also let folks know the date of the next release cut? I feel like we used to do that but we don't any more for some reason.

Also maybe change "7" to "at least 7".

Reply:
> do we want to also let folks know the date of the next release cut? I feel like we used to do that but we don't any more for some reason.

I think that's a good idea. Should we do it in the same message which sends out the release updates?

> Also maybe change "7" to "at least 7".

Done.

Reply:
To your first question, I think a separate message may be better.

Reply:
Done.